test/build/base-url-homepage/base-url-homepage.test.js

```diff
@@ -1,26 +1,40 @@
 const path = require('path');
 const cheerio = require('cheerio');
-const {setupBuildTest, readFiles} = require('../../test-utils');
+const snowpack = require('../../../snowpack');
 
-const cwd = path.join(__dirname, 'build');
+const TEST_ROOT = __dirname;
+const TEST_OUT = path.join(__dirname, 'build');
+let result;
 
-let files = {};
+function getFile(id) {
+  return result[path.resolve(TEST_OUT, id)].contents;
+}
 
 describe('packageManifest.homepage', () => {
-  beforeAll(() => {
-    setupBuildTest(__dirname);
-
-    files = readFiles(cwd);
+  beforeAll(async () => {
+    const [, config] = snowpack.createConfiguration({
+      root: TEST_ROOT,
+      mount: {
+        [path.resolve(TEST_ROOT, './public')]: '/',
```

Comment:
+1 for creating a fresh config object in every test like you do here, and not refactoring this setup into some helper. It's more clear what each test is testing imo.

Reply:
Would this be okay?

```javascript
getBuildResult(result, TEST_OUT, './index.html');
```

The function has to take in the `result` object and the `TEST_OUT` param as they might be different for each test. Or maybe I should curry the function to take the first 2 params on beforeAll?

Reply:
Tbh I though of doing it but then I had the same thought, it's easier to understand what's happening when you see everything in the test file. A helper would've looked more complicated + you'd still have to pass down lots of params for every test to config it which would result in even more unnecessary abstraction.

Reply:
I could create a helper eventually if it becomes useful to have one in the future.